test/eth/TransactionManager.spec.js

```diff
@@ -45,7 +34,12 @@ beforeEach(async () => {
 test('reuse the same web3 and log service in test services', () => {
   expect(services.contract.manager().isConnected()).toBe(true);
   expect(services.txMgr.manager().isConnected()).toBe(true);
-  expect(services.txMgr.get('web3')).toBe(services.contract.get('web3'));
+
+  // Removing _web3 object as blockTrackerTimestamp varies based on
+  // milli/nanoseconds between code execution
+  const _tx = delete { ...services.txMgr.get('web3')._web3 };
```

Comment:
I don't think this does what you want it to do... the return value of `delete` is `true`

Reply:
Reverting back to the original test seems to now work.